aws-auth/iac/lambda.tf

```diff
@@ -0,0 +1,64 @@
+data "aws_iam_policy_document" "lambda" {
+  statement {
+    effect  = "Allow"
+    actions = ["sts:AssumeRole"]
+    principals {
+      type        = "Service"
+      identifiers = ["lambda.amazonaws.com"]
+    }
+  }
+}
+
+data "aws_iam_policy" "lambda" {
+  name = "AWSLambdaBasicExecutionRole"
+}
+
+resource "aws_iam_role" "lambda" {
+  name                = "aws-auth"
+  assume_role_policy  = data.aws_iam_policy_document.lambda.json
+  managed_policy_arns = [data.aws_iam_policy.lambda.arn]
+}
+
+resource "aws_ecr_repository" "app-repo" {
+  name = "aws-auth"
+}
+
+resource "ko_build" "image" {
+  repo        = aws_ecr_repository.app-repo.repository_url
+  importpath  = "github.com/chainguard-dev/enforce-events/aws-auth/cmd/app"
```

Comment:
Repo has been renamed

Reply:
The importpath hasn't yet though. I can do that in another PR and update these all at once.